spec/features/edition_edit_spec.rb

```diff
@@ -267,7 +267,7 @@ def reorder_parts(index, new_index)
     click_navbar_button "Save"
 
     @edition.reload
-    expect(@edition.minor_update).to eq(false)
+    expect(@edition.is_minor_update?).to eq(false)
```

Comment:
```suggestion
    expect(@edition.update_type).to eq("minor")
```

Minor: the rest of the tests check the value directly.